packages/odie-client/src/constants.tsx

```diff
@@ -79,11 +79,22 @@ export const getOdieThirdPartyMessageContent = (): string =>
 		__i18n_text_domain__
 	) }`;
 
-export const getOdieEmailFallbackMessageContent = (): string =>
-	`${ __(
-		"I’m sorry, our human chat support is down for maintenance, but I'm here and ready to assist.",
-		__i18n_text_domain__
-	) } \n\n ${ __( 'What can I help you with?', __i18n_text_domain__ ) }`;
+export const getOdieEmailFallbackMessageContent = ( {
+	isBlockedFromChat = false,
+}: { isBlockedFromChat?: boolean } = {} ): string => {
```

Comment:
Probably easier to make it an argument instead.

```suggestion
export const getOdieEmailFallbackMessageContent = ( isBlockedFromChat = false ): string => {
```

Reply:
Will do.
In cases like this I always think of scenarios where the arg list grows with more booleans and makes me uneasy :D